app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,107 @@
+<head>
+    <title>Recipe <%= @recipe.name %></title>
+  </head>
+  <body>
+    <h1 class="recipe-title"><%= @recipe.name %></h1>
+    <div>
+      <% if @recipe.preparation_time >= 60 && @recipe.cooking_time >= 60 %>
+      <p>
+        Preparation time: <%= @recipe.preparation_time / 60 %>:<%=
+        @recipe.preparation_time % 60 %> hour
+      </p>
+      <p>
+        Cooking time: <%= @recipe.cooking_time / 60 %>:<%= @recipe.cooking_time
+        % 60 %> hour
+      </p>
+      <% else %>
+      <p>Preparation time: <%= @recipe.preparation_time %> mins</p>
+      <p>Cooking time: <%= @recipe.cooking_time %> mins</p>
+      <% end %> <%= @recipe.description %> <% if @recipe.public %>
+      <p>Public Recipe</p>
+      <% else %>
+      <p>Private Recipe</p>
+      <% end %>
+      <% if current_user == @recipe.user %>
+        <%= form_for @recipe, url: recipe_path(@recipe), method: :put do |f| %>
+          <%= f.hidden_field :public, value: !@recipe.public ? true : false%>
+          <%= f.submit @recipe.public ? "Make Private" : "Make Public", class: 'toggle-button' %>
+        <% end %>
+      <% end %>
+
+      <h2 class="text-2xl font-bold mb-4">Ingredients</h2>
+      <table class="w-full table-auto">
+        <!-- Table Headers -->
+        <thead>
+          <tr>
+            <th class="px-4 py-2">Food</th>
+            <th class="px-4 py-2">Quantity</th>
+            <th class="px-4 py-2">Value</th>
+            <th class="px-4 py-2">Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @recipe.recipe_foods.each do |recipe_food| %> <% if
+          recipe_food.food %>
+          <tr>
+            <td class="border px-4 py-2"><%= recipe_food.food.name %></td>
+            <td class="border px-4 py-2">
+              <%= recipe_food.quantity %> <%= recipe_food.food.measurement_unit
+              %>
+            </td>
+            <td class="border px-4 py-2">$<%= recipe_food.food.price %></td>
+            <td class="border px-4 py-2">
+              <a href="#" class="text-blue-500">Modify</a> |
+              <a href="#" class="text-red-500">Remove</a>
+            </td>
```

Comment:
| @cancelei Design|
|-|
| ![image](https://github.com/cancelei/recipe-in-rails/assets/93039359/d090d025-41da-4a8d-b642-d0ffac4a2051) |
|Given Wireframe|
| ![image](https://github.com/ctokoli/RoR-Recipe-app/assets/93039359/41f67813-84b7-4c2c-b1ce-b3ceb1fb4e19) |

- [ ] Great work so far 😊 but I guess you are supposed to add a switch instead of a `Make Public/Private` button for switching the Public/Private status of the recipe 🤔 which you can see in the given wireframe to make your app more professional 💯
- [ ] It would be nice if you align your **Add Ingredient** and **Public/Private Switch**(which you would make now) on the right side of the page to follow the project requirements 👍
- [ ] Great work on adding the `Remove` Button but I guess you misunderstood the change requested by the previous reviewer which was to make the user able to delete their own created content and he highlighted recipe_foods :thinking: Kindly make this button work so the user would be able to delete his own created food_recipes to make your app work as expected :+1: